wasm/tests/graph.rs

```diff
@@ -60,8 +60,8 @@ fn test_implied_edge_rules_reverse_direction() {
         .int_get_edge_by_name(&"1".to_string(), &"root".to_string(), &"up".to_string())
         .unwrap();
 
-    assert_eq!(up_edge_1.weight().implied, true);
-    assert_eq!(up_edge_2.weight().implied, true);
+    assert_eq!(up_edge_1.weight().explicit, true);
+    assert_eq!(up_edge_2.weight().explicit, true);
```

Comment:
Maybe I'm misunderstanding (I also don't see the full context of the file, just the diff), but shouldn't `true` now be `false`? For all instances in this file